examples/c_interface/scf.c

```diff
@@ -0,0 +1,43 @@
+#include <stdlib.h>
```

Comment:
what is this code supposed to do? Is 'scf' a good name for it? Please add comments

Reply:
I still do not understand what this file is about. Maybe it should not be added yet?